### DISCUSSION TITLE
GitHub Education application issue: Missing student email option

### DISCUSSION THREAD

**Discussion by XFZAIMFQ** [community/community#60573] [GitHub Education]
### Hi everyone,

 When submitting the GitHub Education application, select the email address required by "Chongqing Institute of Engineering" as **cqie.edu.cn** This is the school teacher education mailbox, not the student education mailbox, the real student education email address is **stu.cqie.edu.cn**, because the school opened the application channel of the student education mailbox this year, so when you apply for GitHub Education now to use the student email and select the school, the application is not approved, and an email reminder is returned." Attendees of this school must verify and use their academic email address." But **stu.cqie.edu.cn** is indeed a student education email, if you see this feedback, please help us add a student education email address that meets the school, help us pass the application, if you need to confirm the information, I can provide supporting information 

@samus-aran

**Comment by BenjaminDecaillet** [comment]
Hello,
same issue with Unidistance switzerland.

![Re-submit_Github](https://github.com/community/community/assets/26813570/98c86e51-a440-43ce-a46a-1205d7093f34)

**Reply by XFZAIMFQ** [reply]
I've sent multiple emails and submitted a ticket to GitHub, but haven't heard anything yet😢

**Comment by samus-aran** [comment]
Hi @XFZAIMFQ 

This has been addressed and I can see you have been since successful with your application. I'll be closing this discussion due to lack of recent activity. 👍

**Reply by XFZAIMFQ** [reply]
In October this year, I found that there were already student email suffixes and successfully applied for them.  Thank you very much for your help.
